Tests/LibWeb/Text/expected/hit_testing/click-outside-of-box-with-hidden-overflow.txt

```diff
@@ -1 +1 @@
-<HTML>
```

Comment:
Commit message: if this is in a separate PR, it should not exist in this PR?

Reply:
Meant a separate commit, sorry.
Fixed the message.